callback.php

```diff
@@ -80,7 +80,7 @@
 
             if ($status == 0 || $status == 1) {
                 $o->setCurrentState(
-                    Configuration::get('BLOCKONOMICS_ORDER_STATUS_0')
+                    Configuration::get('PS_OS_PREPARATION')
```

Comment:
This is uncessary and can be removed. validateOrder is anyway setting status to 'PS_OS_PREPARATION

Reply:
True, I have removed it and pushed the changes.